lib/all_your_base_cases.rb

```diff
@@ -57,16 +57,17 @@ def error_message
 
 class AllYourBaseCase::PreProcessor
   class << self
+    attr_reader :row
+
     def call(row)
       @row = row
 
       row.merge('expected' => expected_value)
     end
 
+    private :row
     private
 
```

Comment:
This is what we were chatting about on Gitter the other day.

In ruby  < 2.3.0 private reader methods made like that cause a warning. In 2.3 that warning is removed. Links to some info about this are in the last commit message.

Thanks for the feedback!